index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./main.css" type="text/css"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet"> 
+    <title>Alef's Portfolio</title>
+</head>
+<body>
+
+    <header>
+        <a href="">
```

Comment:
- [ ] [Optional] I suggest you add value to the href, when you don't have appropriate value for it, you can pass it a hash `#`

Reply:
Hello,
Thank you for your time reviewing my project. 

I just want to point out that t**he project has 5 template options,** not just the one displayed at the description. 
My project is faithfully close to the one on figma **(template #4).** 
Considering this can i have the approval to merge?